ember_debug/adapters/basic.js

```diff
@@ -113,7 +113,7 @@ export default EmberObject.extend({
   */
   connect() {
     return new Promise((resolve, reject) => {
-      $(() => {
+      document.addEventListener('DOMContentLoaded', () => {
```

Comment:
On that recard we're lucky, we can be sure this is only going to run in chrome and firefox.

About creating a separate npm package, I think it would be overkill really. A simple utility function would be good. This inspector is not something that is going to affect people's apps.